charts/stable/bitmagnet/Chart.yaml

```
@@ -35,4 +35,4 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/bitmagnet
   - https://ghcr.io/bitmagnet-io/bitmagnet
 type: application
-version: 1.1.4
+version: 1.2.0
```

```suggestion
version: 1.1.5
```

This is not a new "feature".